cves/CVE-2014-0116.yml

```diff
@@ -45,7 +45,11 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  A fix introduced in a previous patch (2.3.16.2) to secure the getClass() method
+  didn't account for the wildcard operator (*) being used when configuring the
+  cookiesName param. This opened up the possibility for attackers to manipulate
+  the class loader and modify session state.
```

Comment:
I would be a little more general with this description.

Reply:
Thank you for the feedback; I've generalized my description so it doesn't specify the patch, nor the specific methods used. I also simplified the explanation for how the vulnerability might be exploited.